packages/yoshi-flow-editor/src/wrappers/templates/CommonEditorScriptEntry.ts

```diff
@@ -46,6 +46,7 @@ export default t<Opts>`
     )
     .join('\n  ')}
   editorScriptEntry = builder.build();
+  module.exports = editorScriptEntry.default || editorScriptEntry;
```

Comment:
this check is redundant, since `.build()` will never return an object with `default`

What we can do however, is if the original editorScriptEntry (before you reassigend it with the builder.build result)
then return `default` if it exists

```suggestion
if (editorScriptEntry.default) {
  module.exports = editorScriptEntry.default;
} else {
  builder = editorScriptBuilder()
  //...
    
  module.exports = builder.build()
}
```

Reply:
Got it :+1: